app/assets/stylesheets/all/logs.scss

```diff
@@ -1,2 +1,21 @@
 .project-log-fields:not(:first-child) label { display: none; }
-.project-log-fields:first-child .remove-project-link { display: none; }
\ No newline at end of file
+.project-log-fields:first-child .remove-project-link { display: none; }
+
+.custom-button-class {
+  text-align: center;
+  line-height: 1;
+  cursor: pointer;
+  transition: background-color 0.25s ease-out, color 0.25s ease-out;
+  border: 1px solid transparent;
+  border-radius: 3px;
+  padding: 0.85em 1em;
```

Comment:
Could you switch from `em` to `rem` units here? We typically only use `px` or `rem`.